src/seedu/addressbook/ui/TextUi.java

```diff
@@ -125,7 +125,7 @@ public void showResultToUser(CommandResult result) {
         if (resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(result.getFeedbackToUser(), DIVIDER);
```

Comment:
Correct! All outsider references need to go through the getter method to access information.